spec/unit/hanami/helpers/form_helper_spec.rb

```diff
@@ -2427,6 +2427,34 @@
         expect(actual).to include(%(<select name="book[store]" id="book-store">\n<option value="it" selected="selected">Italy</option>\n<option value="us">United States</option>\n</select>))
       end
     end
+
+    describe 'with nil as a value' do
+      let(:option_values) { Hash['Italy' => 'it', 'United States' => 'us', 'N/A' => nil] }
+
+      it "doesn't set options as selected" do
```

Comment:
This should probably be `it 'sets nil option as selected'`

Reply:
ups, you're right, let me fix it.